components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -108,38 +107,35 @@ bool BraveContentSettingsAgentImpl::IsScriptTemporilyAllowed(
   // Check if scripts from this origin are temporily allowed or not.
   // Also matches the full script URL to support data URL cases which we use
   // the full URL to allow it.
-  return base::Contains(
-      temporarily_allowed_scripts_, script_url.GetOrigin().spec()) ||
-    base::Contains(temporarily_allowed_scripts_, script_url.spec());
+  return base::Contains(temporarily_allowed_scripts_,
+                        script_url.GetOrigin().spec()) ||
+         base::Contains(temporarily_allowed_scripts_, script_url.spec());
 }
 
 void BraveContentSettingsAgentImpl::BraveSpecificDidBlockJavaScript(
     const base::string16& details) {
   Send(new BraveViewHostMsg_JavaScriptBlocked(routing_id(), details));
 }
 
-bool BraveContentSettingsAgentImpl::AllowScript(
-    bool enabled_per_settings) {
+bool BraveContentSettingsAgentImpl::AllowScript(bool enabled_per_settings) {
   // clear cached url for other flow like directly calling `DidNotAllowScript`
   // without calling `AllowScriptFromSource` first
   blocked_script_url_ = GURL::EmptyGURL();
 
   blink::WebLocalFrame* frame = render_frame()->GetWebFrame();
-  const GURL secondary_url(
-      url::Origin(frame->GetDocument().GetSecurityOrigin()).GetURL());
+  const GURL secondary_url(url::Origin(frame->GetSecurityOrigin()).GetURL());
```

Comment:
this happens during the page destruction, for some reason `frame->GetDocument().GetSecurityOrigin()` is already NULL